html/pfappserver/lib/pfappserver/Model/Interface.pm

```diff
@@ -179,8 +178,8 @@ sub down {
     }
 
     # Disable interface using "ip"
-    my $cmd = sprintf "sudo ip link set %s down", $interface;
-    eval { $status = pf_run($cmd) };
+    #    my $cmd = sprintf "sudo ip link set %s down", $interface;
```

Comment:
remove please

Reply:
@jrouzierinverse please remove so we can merge